src/backend/connector.nim

```diff
@@ -0,0 +1,53 @@
+import options, logging
+import json, json_serialization
+import core, response_type
+
+from gen import rpc
+
+const
+  EventConnectorSendRequestAccounts* = "connector.sendRequestAccounts"
+
+type RequestAccountsAcceptedArgs* = ref object of RootObj
+  requestID* {.serializedFieldName("requestId").}: string
+  account* {.serializedFieldName("account").}: string
+  chainID* {.serializedFieldName("chainId").}: uint
+
+type RejectedArgs* = ref object of RootObj
+  requestID* {.serializedFieldName("requestId").}: string
+
+rpc(requestAccountsAccepted, "connector"):
+  args: RequestAccountsAcceptedArgs
+
+rpc(requestAccountsRejected, "connector"):
+  args: RejectedArgs
+
+proc isSuccessResponse(rpcResponse: RpcResponse[JsonNode]): bool =
+  return rpcResponse.error.isNil
+
+proc requestAccountsAcceptedFinishedRpc*(requestID: string, account: string, chainID: uint): bool =
+  try:
+    var args = RequestAccountsAcceptedArgs()
+
+    args.requestID = requestID
+    args.account = account
+    args.chainID = chainID
+
+    let rpcRes = requestAccountsAccepted(args)
+
+    return isSuccessResponse(rpcRes)
+
+  except Exception as e:
+    error "requestAccountsAcceptedFinishedRpc failed: ", "msg", e.msg
+    return false
```

Comment:
Not a huge deal, but you inverted the way we usually do. The service is the one that does processing and the `try/catch` and `backend` usually only does the sending to status-go.

Reply:
I see. In this case, the service doesn't have much to do since the operation is trivial. However, I believe the try/catch block should be closer to where the exception might happen right? In this case putting the exception might mean the expception will buble up to the service before it's caught? Or you meant the try/catch blocks should be in both places (service and connector_backend)?

Reply:
Usualyl we only put the `try/catch` in the service and let the exceptions bubble up like you said.

Reply:
Updated thanks